packages/devtools_app/lib/src/screens/memory/shared/heap/heap.dart

```diff
@@ -194,19 +204,27 @@ class ObjectSet extends ObjectSetStats {
   static ObjectSet empty = ObjectSet()..seal();
 
   final objectsByCodes = <IdentityHashCode, AdaptedHeapObject>{};
+  final excludedFromRetained = <IdentityHashCode>{};
```

Comment:
nit: is this "excludedFromRetained" or "alreadyIncludedInRetained" or similar?

Reply:
renamed